src/views/Home/components/DaysLeft/index.js

```diff
@@ -55,6 +55,9 @@ export default class DaysLeft extends Component {
         datasets: [{ data: [pastDays, daysleft], backgroundColor: [gordonColors.primary.blue] }],
         labels: ['Days Finished', 'Days Remaining'],
       };
+      const options = {
```

Comment:
Possibly change options to something more descript? Looks fine other than that though!